### PR TITLE
Comms Engagement Playbook

### PR DESCRIPTION
**Pull request by candyyap-gvt**

### CODE REVIEW

**Review by kuramochi-coder — COMMENTED** [review]
image
might want to edit the description portion

image
within the resources folder, the overview.html should be renamed to index.html as shown, else the original sidenav selection will not work

Have tested the links to resources, all working fine

**Review by teochengyong — CHANGES_REQUESTED** [review]
Overall looks good, please help to remove unused codes and comments.
- There is a description missing in the communities page
http://localhost:4000/communities/index.html
image

- Added a PR here https://github.com/GovTechSG/developer.gov.sg/pull/1093 to remove the use of inline css.

File: _data/communities.yml [hunk 1]
```diff
@@ -38,3 +39,10 @@
   description: >
     GovBuy is an initiative from the Government to offer engineering, design or
     writing gigs to Singapore citizens and Permanent Residents
+
+- name: Building a Community
+  description: >
+    insert a description
```
[the remark this answers is not in the document]
Comment (candyyap-gvt) [reply _data/communities.yml]: done

File: _includes/community-resources-listing.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  <div class="sgds-card">
+    <div class="sgds-card-content">
+      <h4>Dos and Don'ts</h4>
+      <p>
+        This <a href="/asset/files/Community playbook.pdf" target="_blank" rel="noopener noreferrer nofollow">playbook</a>
+        provides...
+      </p>
+    </div>
+  </div>
+</div>
```
[the remark this answers is not in the document]
Comment (candyyap-gvt) [reply _includes/community-resources-listing.html:32]: removed

File: collections/_communities/building-a-community/resources/interview-with-mindy-lim.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+We are driven by data, and we are avid fans of feedback metrics to measure our success. A key marker of success for us is the tracking of numbers at our events – whether there is an increase in the number of attendees, retention rates for our community Meetup, and tournament participation numbers. 
+
+<p style="text-align: center;"><i>“Progressively, we are looking to also measure success in a qualitative manner such as examining the outcome and impact our initiatives have on an entire agency.”</i></p>
+
+### How do you attract new community members and retain existing ones?
+
+We attract new community members through word of mouth! As the saying goes, “Take care of your employees and they will take care of your customers.” Similarly, when you treat your community members right, they will be your best advocates. 
+
+<p></p>
```
[the remark this answers is not in the document]
Comment (candyyap-gvt) [reply collections/_communities/building-a-community/resources/interview-with-mindy-lim.md]: i assume u meant the <p></p>? nope not needed. Thanks

File: collections/_communities/building-a-community/resources/interview-with-mindy-lim.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+</div>
+<!--<div>
+  <div style="float: left; margin-right: 20px;">
+    <img src="/assets/img/building-a-community-interview-mindy-lim.png" alt="Picture of Mindy Lim">
+  </div>
+  <div>
+    <p>Mindy Lim is a Community Manager under the Data Science & Artificial Intelligence Division (DSAID) in GovTech. As part of the team’s overall capability development initiatives to upskill public officers in all things Data Science and AI, she leads the Data Science Connect (DSC) and Data Arcade Tournament (DAT) events together with her team, uplifting agencies in their data transformation journeys. </p>
+  </div>
+</div>
+<p></p>-->
```
[the remark this answers is not in the document]
Comment (candyyap-gvt) [reply collections/_communities/building-a-community/resources/interview-with-mindy-lim.md]: removed

File: collections/_communities/building-a-community/resources/interview-with-min-li.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+<p style="text-align: center;"><i>“If you are starting a new community, do not allow this kind of behaviour to go on for too long or it will hinder your community’s growth and enthusiasm for participation.”</i></p>
+
+### How do you approach difficult members?
+
+As the admin team keeps enforcing content guidelines, members begin to get a hang of them after a while and model the behaviour. It is wonderful how some of our members take the initiative to step in and respond when someone is getting out of hand. It defuses the situation most of the time. We did have to ban a few people for repeated violations and spammers as well.
+
+<p style="text-align: center;"><i>“Another tip is to have a separate group for the admin team to agree upon how to manage a difficult person or situation. This also ensures that you will come across as a coherent unit. Similarly, keeping each other in the loop is also very important in making decisions so I highly recommend setting up a separate group chat for community leaders to discuss admin matters.”</i></p>
+
+<p></p>
```
[the remark this answers is not in the document]
Comment (candyyap-gvt) [reply collections/_communities/building-a-community/resources/interview-with-min-li.md]: ok

File: collections/_communities/building-a-community/resources/interview-with-min-li.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+  <div style="float: left; margin-right: 20px;">
+    <img src="/assets/img/building-a-community-interview-min-li.png" alt="Picture of Min Li">
+  </div>
+  <div>
+    <p>Min Li is the founder of DesignSG, a Telegram community for designers in Singapore. She has been a mentor for aspiring UX designers at social enterprise, Hatch, and GovTech’s ‘Girls in Tech’ programme. She aims to build a library of mentors across the design fields in DesignSG, to eventually run mentorship programmes for designers in Singapore.</p>
+	
+	<p>Min Li majored in communication at Monash University and completed her Masters in Industrial Design in Milan, Italy. After a hodgepodge of experiences in various start-ups ranging from hardware, healthtech, proptech, edtech, and teaching kids to code, she is now at GovTech, working with JTC's Smart District Division.</p>
+  </div>
+</div>
+<p></p>-->
```
[the remark this answers is not in the document]
Comment (candyyap-gvt) [reply collections/_communities/building-a-community/resources/interview-with-min-li.md]: removed

File: collections/_communities/building-a-community/resources/overview.html [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+  <div class="sgds-card">
+    <div class="sgds-card-content">
+      <h4><a href="interview-with-melvin-tan-and-michael-cheng" target="_blank" rel="noopener noreferrer nofollow">Interview with Melvin Tan and Michael Cheng</a></h4>
+ 	  <p>02 Nov 2021</p>
+    </div>
+  </div>
+</div>
+
+
+<!--{% include community-resources-listing.html %}-->
```
[the remark this answers is not in the document]
Comment (candyyap-gvt) [reply collections/_communities/building-a-community/resources/overview.html]: done

**Review by kuramochi-coder — APPROVED** [review]
Looks good, sgds css classes are used as far a possible and content loads correctly

**Pull request merged**